### DISCUSSION TITLE
Simple State Level Energy Model Input Data

### DISCUSSION THREAD

**Discussion by NicholasHRoy** [catalyst-cooperative/pudl#3452] [Help Me!] [answered]
Hi, 

We are trying to convert some of our data to open source so we can hopefully move a capacity expansion model to be fully available for public use. I have been trying to navigate PUDL for some time and find most of the things I need but am struggling for others. Is there anywhere I can find: 

- Fixed Operations and Maintenance Costs
- Variable Operations and Maintenance Costs
- Emissions rates (I imagine this can be imputed)

I think I've identified the following locations for things:

- capacity
- location
- capex
- fuel costs
- generation
- transmission constraints

For context this model will have 49 nodes for the lower 48 states + DC.  Some aggregation will be done of course. Looking at the PowerGenome project, it seems like everything I am looking for should be here, but the question is really where. Thanks.

**Comment by zaneselvans** [comment, marked as answer]
Hi there, glad you've found some of our data useful!

* If you want plant-specific Fixed & Variable O&M costs, the only place you could look in the current PUDL data is the FERC Form 1 large steam plants table: [`out_ferc1__yearly_steam_plants_sched402`](https://data.catalyst.coop/pudl/out_ferc1__yearly_steam_plants_sched402). It contains a number of columns we've labeled with `capex` and `opex`. Unfortunately, we haven't been able to find a clear line between fixed and variable costs. We tried doing some regressions to identify which cost categories scaled with generation, and which scaled with plant capacity, and even when binning by technology type and plant vintage, we didn't get any real clear result. It could be that the sample size (especially with the binning) just wasn't large enough, or it could be that different utilities are reporting different expenses in different categories.  @cmgosnell knows more about all this data than anyone else.
* While it's not technically O&M, the ongoing flow of capital investments that are required to keep the plants running but that aren't part of any major plant upgrade are also recorded in the steam plants table, and in general that constant flow of CapEx ends up being the predominant type of CapEx, and in a retirement vs. non-retirement decision, it seems like this would be a useful per-plant number to have around.
* Also note that unfortunately, the FERC Form 1 only covers a portion of the generation fleet, generally leaving out any public power or co-op owned resources, and some subset of the competitive merchant fleet. And also, it's quite challenging to link the FERC 1 "plants" to their corresponding collections of EIA generators and/or ownership slices.  We have a way of doing this, but it's a little involved right now.
* For generic fixed & variable O&M numbers by technology type or plant vintage, we've historically used either the NREL ATB or EIA's NEMS model inputs. Hopefully we'll be adding an ATB derived table to PUDL soon for easier access.
* When you say emissions rates, are you thinking just GHG or other traditional pollution? If it covers the generators you care about, you could calculate some averages based on the EPA CEMS hourly data, which we publish as a single Parquet file. We also have a crosswalk that addresses many of the small inconsistencies between the IDs used in that EPA data, and the EIA plants/units it refers to.
* Where have you found the transmission constraints?  That's often a big hole that people are frustrated at being unable to fill.

**Reply by NicholasHRoy** [reply]
Hi Zane, this is really helpful. I realized shortly after this FERC form 1 has the partial answer, but it is good to know ahead of time that it can be an unfruitful rabbit hole. The level of our model is 49 nodes for the contiguous US so for transmission constraints I think we really plan to just look back at historical US transmission between states and consider it a maximum. Understanding that isn't exactly how the grid works, it might do for our pipeflow style model. Could you point me to the EIA NEMS outputs you mentioned? That could be really useful for us. Thanks for the responses on the other points, very helpful.

**Reply by zaneselvans** [reply]
You can grab the NEMS inputs from here https://www.eia.gov/outlooks/aeo/info_nems_archive.php

I think you may need to request the decryption key through email, and you'll need to root around in the contents of the zipfile to find them. EIA has historically been paranoid about people using this data for purposes other than running NEMS.  But everybody does anyway.  Also note that NEMS is currently undergoing a huge overhaul, and will probably look very different in 2025.
